dashboard/package.json

```diff
@@ -141,7 +142,6 @@
     "lint-staged": "^10.2.11",
     "mock-socket": "^9.0.3",
     "moxios": "^0.4.0",
-    "node-sass-chokidar": "^1.3.3",
```

Comment:
you need to add the `sass` devdep, no?

Reply:
Absolutely right. I added the `--dev` flag when doing the clarity stuff, but totally forgot about it when extracted the change for this PR :S
Thank you!!